<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeEMS_LoaderClass</class>
 <widget class="QWidget" name="FreeEMS_LoaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeEMS_Loader</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>FreeEMS_Picture_5.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushConnect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushLoad">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Firmware</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>761</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRip">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Rip Firmware</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TX</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>55</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushGo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Device:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDevice">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkRip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Rip Before Load</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkVerify">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Verify Load</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Flash Type</string>
     </property>
     <widget class="QRadioButton" name="radFlashType">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>S12XDP512</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Advanced Settings</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop bits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBaud">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Data bits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDataBits">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>78</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboStopBits">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboParity">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>78</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSoft">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Software</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkHard">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Hardware</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Handshake</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource-root.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushConnect</sender>
   <signal>clicked()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRip</sender>
   <signal>clicked()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>rip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal2()</signal>
  <slot>test()</slot>
  <slot>slot1()</slot>
  <slot>connect()</slot>
  <slot>rip()</slot>
  <slot>getFileName()</slot>
 </slots>
</ui>
